### DISCUSSION TITLE
Integrate SVG symbol in QGIS projects

### DISCUSSION THREAD

**Discussion by sjib** [teksi/Home#107] [Ideas]
For certain symbols the svg format is a good choice and it should be discussed where we should change to svg symbols for the TEKSI projects.

The question then is how to we access the svg symbols?

If found this documentation:

Einbetten des SVG-Symbols in die QGIS-Projektdatei
Es ist umständlich, die SVG-Symbole jeweils mit dem Projekt dazu zu liefern. Ausserdem könnte es Probleme mit dem Pfad geben."
Deshalb betten wir das SVG ins Projekt ein.

Rechtsklick auf den Layer > Eigenschaften... > Symbolisierung > gehe zum SVG-Symbol > wähle *.svg aus und wähle “Datei einbetten”
Speichere das Projekt. Nun können wir die SVG-Datei aus dem Ordner entfernen.
![svg_in_qgs_einbetten](https://github.com/teksi/Home/assets/1265865/8cc4f90d-987b-4d6c-8db6-81d5bf166432)

Intégrer le symbole SVG dans le fichier de projet QGIS Il est compliqué de livrer les symboles SVG avec le projet. De plus, il pourrait y avoir des problèmes avec le chemin", c'est pourquoi nous intégrons le SVG dans le projet.

Clic droit sur le layer > Propriétés... > Symbolisation > aller sur le symbole SVG > sélectionner *.svg et choisir "Intégrer le fichier" 
Enregistrer le projet. Nous pouvons maintenant supprimer le fichier SVG du dossier.

Source https://download.opengis.ch/public/courses/HSR_advanced_sep2023/documents/drehbuch.pdf page 10

Does anyone uses this like that? @urskaufmann @varrieta @ponceta @Fabien-Cattin @cymed @jpdupuy 

What are the pro / con of this approach?

**Comment by cymed** [comment]
We use it in qgis projects that we have to send to other offices. It works, but the project size increases and it is inefficient when you want to use one svg multiple times. Ideally we'd create a subfolder in the user profile's svg abd download the svg from github on the plugin installation

**Comment by urskaufmann** [comment]
Pro: it works with Qfield. If SVG symbols are not integrated, there are just "?" if you open a project in Qfield

**Comment by sjib** [comment]
[SVG symbol library for District Heating](https://github.com/teksi/district_heating/issues/31)
